services/core-api/app/api/mines/permits/permit_conditions/models/standard_permit_conditions.py

```diff
@@ -27,11 +26,47 @@ class StandardPermitConditions(SoftDeleteMixin, AuditMixin, Base):
     parent_standard_permit_condition_id = db.Column(
         db.Integer, db.ForeignKey('standard_permit_conditions.standard_permit_condition_id'))
     display_order = db.Column(db.Integer, nullable=False)
-    sub_conditions = db.relationship(
+    all_sub_conditions = db.relationship(
         'StandardPermitConditions',
         lazy='joined',
+        order_by='asc(StandardPermitConditions.display_order)',
         backref=backref('parent', remote_side=[standard_permit_condition_id]))
 
+    def __repr__(self):
+        return '<StandardPermitConditions %r, %r>' % (self.standard_permit_condition_id,
+                                                      self.standard_permit_condition_guid)
+
+    def __str__(self):
+        return f'<StandardPermitConditions> standard_permit_condition_id: {self.standard_permit_condition_id}, standard_permit_condition_guid: {self.standard_permit_condition_guid}, condition: {self.condition}, condition_category_code: {self.condition_category_code}, condition_type_code: {self.condition_type_code}, notice_of_work_type: {self.notice_of_work_type}, parent_standard_permit_condition_id: {self.parent_standard_permit_condition_id}, display_order: {self.display_order}, all_sub_conditions: {self.all_sub_conditions}'
+
+    @hybrid_property
+    def sub_conditions(self):
+        return [x for x in self.all_sub_conditions if x.deleted_ind == False]
+
+    # The FE is expecting the same keys are permitConditions
```

Comment:
Can we remove this comment? I don't understand it and am not sure if it's helpful. 😜